packages/search/tests/integration/components/navi-search-result/definition-test.js

```diff
@@ -77,19 +84,62 @@ module('Integration | Component | definition', function(hooks) {
 
     await render(hbs`<NaviSearchResult::Definition @data={{this.result.data}} />`);
 
-    assert.dom('.navi-search-result__definition.result_element').exists({ count: 1 }, 'One result is displayed');
+    assert.dom('.navi-search-definition-result').exists({ count: 1 }, 'One result is displayed');
 
-    const definitionTitles = findAll('.navi-search-result__definition-name').map(el => el.textContent.trim());
-    const definitionDescriptions = findAll('.navi-search-result__definition-description').map(el =>
-      el.textContent.trim()
+    assert.deepEqual(
+      findAll('.navi-search-definition-result__item-name').map(el => el.textContent.trim()),
+      ['Page Views'],
+      'definition titles are shown correctly'
     );
-    const expectedDefinitionTitles = ['Page Views'];
-    const expectedDefinitionDescriptions = ['The number of views of a page.'];
-    assert.deepEqual(definitionTitles, expectedDefinitionTitles, 'definition titles are shown correctly');
     assert.deepEqual(
-      definitionDescriptions,
-      expectedDefinitionDescriptions,
+      findAll('.navi-search-definition-result__item-description').map(el => el.textContent.trim()),
+      ['The number of views of a page.'],
       'definition descriptions are shown correctly'
     );
   });
+
+  test('Multiple datasources', async function(assert) {
+    assert.expect(4);
+
+    const data = [
+      {
+        id: 'pageViews',
+        name: 'Page Views',
+        description: 'The number of views of a page.',
+        source: 'dummy'
+      },
+      {
+        id: 'impressions',
+        name: 'Impressions',
+        description: 'Number of times a user saw the ad.',
+        source: 'blockhead'
+      }
+    ];
+    const result = {
+      component: 'navi-search-result/definition',
+      title: 'Definitions',
+      data
+    };
+    set(this, 'result', result);
+
+    await render(hbs`<NaviSearchResult::Definition @data={{this.result.data}} />`);
```

Comment:
```suggestion
    this.data = [
      {
        id: 'pageViews',
        name: 'Page Views',
        description: 'The number of views of a page.',
        source: 'dummy'
      },
      {
        id: 'impressions',
        name: 'Impressions',
        description: 'Number of times a user saw the ad.',
        source: 'blockhead'
      }
    ];
    
    await render(hbs`<NaviSearchResult::Definition @data={{this.data}} />`);
```
Seems like the rest of the result object is being declared but not used.